### ISSUE TITLE
False negative on Style/FormatString?

### ISSUE THREAD

**Issue by kakutani** [rubocop/rubocop#1512]
Rubocop(master) looks make false nagative against typical sprintf/format cases.

[Some example cases](https://github.com/kakutani/rubocop/commit/259ea9dd0a89547e8ca7abbc0ec8d6d6758dd8c0#diff-1)(picked from [ruby-doc.org](http://www.ruby-doc.org/core-2.1.5/Kernel.html#method-i-sprintf)) will [fail tests](https://gist.github.com/kakutani/3324943902f8272c671c).

[This patch](https://github.com/kakutani/rubocop/commit/0513a2f514ef1f2e2d3c856b71da33ca2a950244#diff-0) will make the test suite is passing:

``` diff
diff --git a/lib/rubocop/cop/style/format_string.rb b/lib/rubocop/cop/style/format_string.rb
index e75095a..37c12df 100644
--- a/lib/rubocop/cop/style/format_string.rb
+++ b/lib/rubocop/cop/style/format_string.rb
@@ -31,13 +31,13 @@ def offending_node?(node)
         end

         def format_method?(name, node)
-          receiver, method_name, args = *node
+          receiver, method_name, *args = *node

           # commands have no explicit receiver
           return false unless !receiver && method_name == name

           # we do an argument count check to reduce false positives
-          args && args.children.size >= 2
+          args.size >= 2
         end

         def format?(node)
```

I'm not sure why `format(something, a, b)` registers an offense in spec but `format("%d %04x", 123, 123)` does not.

It would be great if you guys look into it.
